### ISSUE TITLE
[FEATURE] change `login anonymously`

### ISSUE THREAD

**Issue by Dun-sin** [Dun-sin/Whisper#119]
### Description

Make the `login anonymously` button to just a clickable text

- [ ] remove the background
- [ ] add an underline on hover

new:
![image](https://user-images.githubusercontent.com/78784850/195779062-0aa4e42d-eaa4-4b71-94cd-862524fb40a2.png)

### Screenshots

Old:
![image](https://user-images.githubusercontent.com/78784850/195778784-c6fc84bb-7c40-4955-b57c-86e8b370afbf.png)

**Comment by yhoungdev** [comment]
.token

**Comment by yhoungdev** [comment]
will handle this

**Comment by Divyesh1511** [comment]
Can you assign me this @Dun-sin?

**Comment by Dun-sin** [comment]
> Can you assign me this @Dun-sin?

Definitely, thanks for wanting to contribute, Make sure you follow the rules [here](https://github.com/Dun-sin/Whisper/blob/main/CONTRIBUTING.md#rules), else your PR will not be accepted
Good Luck

**Comment by Divyesh1511** [comment]
Hey, @Dun-sin I am unable to run the project on my work desk can you please just help me out

**Comment by yhoungdev** [comment]
> Hey, @Dun-sin I am unable to run the project on my work desk can you please just help me out

whats your Node version. if its Version 18 then you need to downgrade to 16 or 17 

you can use NVM to handle that

**Comment by Dun-sin** [comment]
> Hey, @Dun-sin I am unable to run the project on my work desk can you please just help me out 
> 
> 

What happened when you tried npm start

**Comment by Divyesh1511** [comment]
> > Hey, @Dun-sin I am unable to run the project on my work desk can you please just help me out
> 
> What happened when you tried npm start

while doing the npm install only it throws an error

**Comment by Dun-sin** [comment]
> > > Hey, @Dun-sin I am unable to run the project on my work desk can you please just help me out
> > 
> > What happened when you tried npm start
> 
> while doing the npm install only it throws an error

Figured, what's the error

**Comment by Divyesh1511** [comment]
> > Hey, @Dun-sin I am unable to run the project on my work desk can you please just help me out
> 
> whats your Node version. if its Version 18 then you need to downgrade to 16 or 17
> 
> you can use NVM to handle that

Ok I guess this might be the issue let me try once

**Comment by yhoungdev** [comment]
> > > Hey, @Dun-sin I am unable to run the project on my work desk can you please just help me out
> > 
> > 
> > whats your Node version. if its Version 18 then you need to downgrade to 16 or 17
> > you can use NVM to handle that
> 
> Ok I guess this might be the issue let me try once

You can paste the issue here or image of it

**Comment by Divyesh1511** [comment]
> > > > Hey, @Dun-sin I am unable to run the project on my work desk can you please just help me out
> > > 
> > > 
> > > What happened when you tried npm start
> > 
> > 
> > while doing the npm install only it throws an error
> 
> Figured, what's the error

Sorry I couldn't get it

**Comment by Divyesh1511** [comment]
@yhoungdev this is what it shows
![Screenshot (772)](https://user-images.githubusercontent.com/75933218/195881359-e9b13a0a-87a2-4dfd-8fce-c5abb6f3451a.png)

**Comment by yhoungdev** [comment]
You got this while trying to npm install right ?

**Comment by Divyesh1511** [comment]
> You got this while trying to npm install right ?

Yes

**Comment by Dun-sin** [comment]
> @yhoungdev this is what it shows
> ![Screenshot (772)](https://user-images.githubusercontent.com/75933218/195881359-e9b13a0a-87a2-4dfd-8fce-c5abb6f3451a.png)
> 

Could be your internet connection, try again

**Comment by Divyesh1511** [comment]
> > @yhoungdev this is what it shows
> > ![Screenshot (772)](https://user-images.githubusercontent.com/75933218/195881359-e9b13a0a-87a2-4dfd-8fce-c5abb6f3451a.png)
> 
> Could be your internet connection, try again

It's perfectly working fine, I guess there is another issue

**Comment by Dun-sin** [comment]
> It's perfectly working fine, I guess there is another issue

Well did you try again?

**Comment by Divyesh1511** [comment]
> > It's perfectly working fine, I guess there is another issue
> 
> Well did you try again?

Yes tried multiple times now I am downgrading my node version and trying once again

**Comment by yhoungdev** [comment]
> > > It's perfectly working fine, I guess there is another issue
> > 
> > Well did you try again?
> 
> Yes tried multiple times now I am downgrading my node version and trying once again

If you are running node 18 you should get already while doing npm install

**Comment by Divyesh1511** [comment]
> > > > It's perfectly working fine, I guess there is another issue
> > > 
> > > 
> > > Well did you try again?
> > 
> > 
> > Yes tried multiple times now I am downgrading my node version and trying once again
> 
> If you are running node 18 you should get already while doing npm install

So you are saying that npm install should work in node 18 too?

**Comment by Dun-sin** [comment]
> So you are saying that npm install should work in node 18 too?
It should not be a problem

**Comment by yhoungdev** [comment]
> > So you are saying that npm install should work in node 18 too?
> It should not be a problem
> 

Became some days back I tried installing npm on this application and I got warning urging to downgrade to Node 16

**Comment by Dun-sin** [comment]
@Divyesh1511 if you aren't able to install this project, let me know

**Comment by Divyesh1511** [comment]
> @Divyesh1511 if you aren't able to install this project, let me know

Yes I am unable to install it

**Comment by Divyesh1511** [comment]
You can assign it to anyone else, really sorry for the inconvenience

**Comment by Dun-sin** [comment]
> > @Divyesh1511 if you aren't able to install this project, let me know
> 
> Yes I am unable to install it

okay then, i don't know what's wrong, so i will just assign this to someone else

**Comment by Divyesh1511** [comment]
> okay then, i don't know what's wrong, so i will just assign this to someone else

Ok

**Comment by yhoungdev** [comment]
Working on the issue

**Comment by yhoungdev** [comment]
This issue has been fixed on #124

**Comment by creative-tutorials** [comment]
On it
